lldb/source/Plugins/Process/Utility/ThreadMemory.h

```diff
@@ -13,14 +13,14 @@
 
 #include "lldb/Target/Thread.h"
 
+/// A memory thread optionally backed by a real thread.
+/// All methods of this class dispatch to the real thread, if it is not null.
+/// Otherwise the methods are no-op.
```

Comment:
This isn't 100% true, the MemoryThread has a different TID from the backing thread (sometimes providing a stable TID is the ONLY job the MemoryThread is doing.